modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -146,6 +146,24 @@ For more information about GPG keys, see xref:client-configuration:gpg-keys.adoc
 {debian} clients can require multiple GPG keys to be installed.
 ====
 
+When synchronizing third-party {debian} repositories, you will need to import the appropriate GPG key.
+If the GPG key is missing, synchronization will fail.
+ 
+Only the metadata is signed, therefore importing a GPG key for the software channel is not needed.
```

Comment:
Not sure what this should mean? Do you want to say that the 3rd Party GPG key is not needed on the Client as the metadata are signed only and Uyuni either do not sign them or with an own GPG key which will be trusted on the clients automatically?

Reply:
Hi @mcalmer,
sorry for the late reply!

The GPG key is missing on the server side (so on Uyuni or SUSE Manager) as otherwise the download will fail. The packages won't be re-sigend by Uyuni.

Reply:
I tried to rephrase the context in commit dcf3b8f8b283fd1930555dc133bde21fc7f26619 - does this help?